src/utils/dom.ts

```diff
@@ -26,6 +26,13 @@ export function getThemeName(el: HTMLElement): "light" | "dark" {
   return closestElementCrossShadowBoundary(el, `.${CSS_UTILITY.darkTheme}`) ? "dark" : "light";
 }
 
+export function getElementStyleDir(el: HTMLElement): Direction {
```

Comment:
I could also just replace `getElementDir` with this logic. But we should refactor all uses of getElementDir to see where we can remove its use and use CSS logical properties instead.